xonsh/shell.py

```diff
@@ -121,54 +121,36 @@ class Shell(object):
         "best": "best",
         "d": "dumb",
         "dumb": "dumb",
-        "ptk": "prompt_toolkit",
-        "ptk1": "prompt_toolkit1",
-        "ptk2": "prompt_toolkit2",
+        "ptk": "prompt_toolkit",  # there's only 1 prompt_toolkit shell (now)
+        "ptk1": "prompt_toolkit",  # allow any old config reference to use it
+        "ptk2": "prompt_toolkit",  # so long as user actually  has ptk2+ installed.
         "prompt-toolkit": "prompt_toolkit",
         "prompt_toolkit": "prompt_toolkit",
-        "prompt-toolkit1": "prompt_toolkit1",
-        "prompt-toolkit2": "prompt_toolkit2",
+        "prompt-toolkit1": "prompt_toolkit",
+        "prompt-toolkit2": "prompt_toolkit",
```

Comment:
I think we should add a ptk3 here as well as make ptk1 issue a warning or similar

Reply:
warn about ptk1 but continue? OK.  User must actually have acceptable ptk installed, just needs to update RC file.

My thinking about no ptk3 is there's no difference between ptk2 and ptk3 (to Xonsh), maybe now's the time to wean users off the idea they have to worry about specific version of ptk?  After all, there's no version flavors of readline.  And who knows what ptk 4 will be? If it's small, Xonsh might slide by on backward compat.  If it's a big change, it might not even be called prompt-toolkit...

Reply:
For me it is more about the semantics of it.  It seems weird to not have a ptk3 if we have ptk1 & ptk2.

Reply:
OK, I'll add it here but make my pitch for using just 'prompt_toolkit' in the doc.